### PR TITLE
Adding kube dns benchmark to presubmit jobs

### PR DESCRIPTION
**Pull request by vamossagar12**

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml [hunk 1]
```diff
@@ -380,6 +380,43 @@ presubmits:
...
+      testgrid-tab-name: kube-dns
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    run_if_changed: ^dns/.*$
+    spec:
+      containers:
+      - args:
+        - --repo=k8s.io/kubernetes=master
+        - --repo=k8s.io/perf-tests=master
```
Comment (mm4tt) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml:400]: Should be 
```
--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
```

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml [hunk 2]
```diff
@@ -380,6 +380,43 @@ presubmits:
           privileged: true # TODO(fejta): https://github.com/kubernetes/kubernetes/issues/76484
 
   kubernetes/perf-tests:
+  - name: pull-perf-tests-dns-benchmarks
```
Comment (mm4tt) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml:383]: Let's name it: `pull-perf-tests-benchmark-kube-dns`

**Comment by vamossagar12** [comment]
> /ok-to-test

made those 2 changes + needed to change the testgrid-tab-name because bazel build had failed the last time:

```
ERROR: /home/prow/go/src/github.com/kubernetes/test-infra/config/tests/testgrids/BUILD.bazel:25:1: Executing genrule //config/tests/testgrids:testgrid_config failed (Exit 1)
2020/09/15 11:21:29 FAIL: could not write config: 1 error occurred:
	* found duplicate name after normalizing: (DashboardTab) kubedns

```

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml [hunk 3]
```diff
@@ -380,6 +380,42 @@ presubmits:
...
 
   kubernetes/perf-tests:
+  - name: pull-perf-tests-benchmark-kube-dns
+    always_run: false
+    skip_report: false
+    max_concurrency: 3
+    branches:
+      - master
+    annotations:
+      testgrid-dashboards: sig-scalability-benchmarks
```
Comment (mm4tt) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml:390]: Good catch with the dashboards. Let's change this as well, it should be in the presubmit dashboard:
```
testgrid-dashboards: presubmits-kubernetes-scalability
```

Comment (vamossagar12) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml]: done.

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml [hunk 4]
```diff
@@ -380,6 +380,42 @@ presubmits:
...
+      testgrid-dashboards: presubmits-kubernetes-scalability
+      testgrid-tab-name: pull-perf-tests-benchmark-kube-dns
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    run_if_changed: ^dns/.*$
+    spec:
+      containers:
+      - args:
+        - --repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
```
Comment (mm4tt) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml:399]: We'll need also 
```
- --repo=k8s.io/release
```

**Review by mm4tt — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**